delay-relay-on.js

```diff
@@ -0,0 +1,23 @@
+const CONFIG = {
+    /**
+     * Choose your device switch ID to be used for turning on the switch after time delay
+     */
+    ID: 0,
+    
+    /**
+     * Choose DELAY time in seconds after which the switch is turned on
+     */
+    DELAY: 5
+}
+
+let timerID;
+
+function turnSwitchOn() {
+    Shelly.call("Switch.Set", {"id": CONFIG.ID, "on": true});
```

Comment:
Move the component type to the config, the user might want to turn on a light or rgb component.